### PR TITLE
metrics: update TiKV metrics

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve? 

Update TiKV metrics

### Check List 

Tests 

 - No code

Related changes

 - Need to cherry-pick to the release branch

### Release note

- No release note

### CODE REVIEW

File: metrics/grafana/ticdc.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "__inputs": [
     {
-      "name": "DS_TEST-CLUSTER",
-      "label": "test-cluster",
```
Comment (amyangfei) [inline metrics/grafana/ticdc.json:5]: no need to update the default label name

File: metrics/grafana/ticdc.json [hunk 2]
```diff
@@ -5741,9 +5852,18 @@
               "format": "time_series",
               "hide": false,
               "intervalFactor": 2,
-              "legendFormat": "tikv-{{instance}}",
+              "legendFormat": "tikv-{{instance}}-total",
               "refId": "A",
               "step": 10
+            },
+            {
+              "expr": "avg(tikv_cdc_region_resolve_status{tidb_cluster=\"$tidb_cluster\", instance=~\"$tikv_instance\"}) by (instance, status)",
```
Comment (overvenus) [inline metrics/grafana/ticdc.json:5860]:
```suggestion
              "expr": "sum(tikv_cdc_region_resolve_status{tidb_cluster=\"$tidb_cluster\", instance=~\"$tikv_instance\"}) by (instance, status)",
```

**Pull request merged**